manual/sphinx/user_docs/advanced_install.rst

```diff
@@ -258,12 +258,28 @@ appropriately.
 OpenMP
 ------
 
-BOUT++ can make use of Single-Instruction Multiple-Data (SIMD)
-parallelism through OpenMP. To enable OpenMP, use the
+BOUT++ can make use of OpenMP parallelism. To enable OpenMP, use the
 ``--enable-openmp`` flag to configure::
 
     ./configure --enable-openmp
 
+OpenMP can be used to parallelise in more directions than can be
+achieved with MPI alone. For example, it is currently difficult to
+parallelise in X using pure MPI if FCI is used, and impossible to
+parallelise at all in Z with pure MPI.
+
+OpenMP is in a large number of places now, such that a decent speed-up
+can be achieved with OpenMP alone. Hybrid parallelisation with both
+MPI and OpenMP can lead to more significant speed-ups, but it
+sometimes requires some fine tuning of numerical parameters in order
+to achieve this. This greatly depends on the details not just of your
+system, but also your particular problem. We have tried to choose
+"sensible" defaults that will work well for the most common cases, but
+this is not always possible. You may need to perform some testing
+yourself to find e.g. the optimum split of OpenMP threads and MPI
+ranks.
```

Comment:
Is it worth mentioning the configure time option to set the schedule?

Reply:
Yes -- I wasn't aware of that when I wrote this!